### PR TITLE
Part3

### PR DESCRIPTION
**Pull request by polypx**

### CODE REVIEW

File: main.cpp [hunk 1]
```diff
@@ -124,10 +147,19 @@ void Tree::swayInTheWind(double windSpeed)
 {
     if(windSpeed < 0)
     {
-        std::cout << "negative wind speed not allowed \n";      
+        std::cout << "negative wind speed not allowed" << std::endl;       
+    } else
```
Comment (matkatmusic) [inline main.cpp]: Curly braces go on their own line with no other code
- [x] revise

File: main.cpp [hunk 2]
```diff
@@ -214,9 +262,14 @@ struct ControlRoom
...
 
+ControlRoom::ControlRoom()
+{
+    std::cout << "ControlRoom being constructed." << std::endl;
+}
+
 void ControlRoom::seatEngineer(std::string engineerName)
 {
-        std::cout << "Today we're enjoying the mixing skills of  " << engineerName;      
+        std::cout << "Today we're enjoying the mixing skills of " << engineerName << std::endl;
```
Comment (matkatmusic) [inline main.cpp]: Fix this indentation.
Code should be indented 4 spaces from the surrounding curly braces
- [x] revise

File: main.cpp [hunk 3]
```diff
@@ -264,49 +324,105 @@ struct LiveRoom
...
+    std::cout << "We have taken delivery of a " << instrumentType.instrument1  << std::endl;     
 }
 
 bool LiveRoom::switchLights()
 {
     lightsCurrentState = !lightsCurrentState;
+    if (lightsCurrentState)
+    {
+        std::cout << "The lights are currently on."  << std::endl; 
+    } else
```
Comment (matkatmusic) [inline main.cpp]: Curly braces go on their own line with no other code
- [x] revise

**Review by matkatmusic — CHANGES_REQUESTED** [review]

File: main.cpp (hunk 1 above)
Comment (polypx) [reply main.cpp]: revised

File: main.cpp (hunk 2 above)
Comment (polypx) [reply main.cpp]: revised

File: main.cpp (hunk 3 above)
Comment (polypx) [reply main.cpp]: revised

**Review by matkatmusic — APPROVED** [review]

**Pull request merged**